common/models/role.js

```diff
@@ -271,6 +306,9 @@ module.exports = function(Role) {
    * @param {Boolean} isInRole True if the principal is in the specified role.
    */
   Role.isInRole = function(role, context, callback) {
+    var self = this;
+    callback = callback || utils.createPromiseCallback();
+    context.registry = Role.registry;
     if (!(context instanceof AccessContext)) {
       context = new AccessContext(context);
     }
```

Comment:
related to comment on having access to local app registry inside AccessContext methods